source/UI.cpp

```diff
@@ -270,6 +271,34 @@ Point UI::GetMouse()
 
 
 
+void UI::PlaySound(UISound sound)
+{
+	string name;
+	switch(sound)
+	{
+		case UISound::NORMAL:
+			name = "ui/click";
+			break;
+		case UISound::SOFT:
+			name = "ui/click_soft";
+			break;
+		case UISound::SOFT_BUZZ:
+			name = "ui/buzz_soft";
```

Comment:
We don't have problems with spaces in sound files in general.
```suggestion
			name = "ui/click soft";
			break;
		case UISound::SOFT_BUZZ:
			name = "ui/buzz soft";
```